apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/StorageClient.UploadObject.cs

```diff
@@ -153,5 +153,51 @@ public abstract partial class StorageClient
         {
             throw new NotImplementedException();
         }
+
+        /// <summary>
+        /// Initiates an upload session, optionally specifying the length of the content to be uploaded.
+        /// The resulting URI can be used with <see cref="ResumableUpload.CreateFromUploadUri"/>.
+        /// </summary>
+        /// <param name="destination">Object to create or update. Must not be null, and must have the name
+        /// and bucket populated.</param>
+        /// <param name="contentLength">The length of the content to upload later. This may be null, in which
+        /// case any length of content may be uploaded. If the value is non-null, it must be strictly positive
+        /// (not zero), and the content uploaded later must be exactly this length.</param>
+        /// <param name="cancellationToken">The token to monitor for cancellation requests.</param>
+        /// <param name="options">Additional options for the upload. May be null, in which case appropriate
```

Comment:
super nit: swap these two? I thought Visual Studio would catch that.

Reply:
Whoops, yes, will do.